src/Components/Login.js

```diff
@@ -1,30 +1,139 @@
 import Button from 'react-bootstrap/Button';
 import Form from 'react-bootstrap/Form';
 import 'bootstrap/dist/css/bootstrap.min.css'
+import { useEffect, useState } from 'react';
+import AdminPanel from './adminPanel';
+import { json, useNavigate  } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
+import Dropdown from 'react-bootstrap/Dropdown';
+
+function Login() {
+
+    const navigate= useNavigate();
+
+     const [Email, setEmail] = useState('');
+     const [Password, setPassword] = useState('');
+     const [Loggedin, SetLoggedin]= useState(false);
+     let selectedvalue;
+
+     const [selectedOption, setSelectedOption] = useState('0');
+
+    const handleOptionChange = (event) => {
+        setSelectedOption(event.target.value);
+        selectedvalue=event.target.value;
+        console.log(selectedvalue);
+    };
+
+
+    //  useEffect(()=>{
+
+    //         return(
+    //             <div>
+    //                 {<AdminPanel/>}
+    //             </div>
+    //         )
+
+    //  },[Loggedin])
```

Comment:
please remove the commented code block if you think, it is no longer needed. If you think you need it future for some reasons, just comment a todo. exm: TODO: implement useEffect()